packages/edit-site/src/components/sidebar-navigation-screen-navigation-menu/index.js

```diff
@@ -43,6 +63,56 @@ export default function SidebarNavigationScreenNavigationMenu() {
 
 	const menuTitle = navigationMenu?.title?.rendered || navigationMenu?.slug;
 
+	const handleSave = async () => {
+		saveEditedEntityRecord( 'postType', postType, postId );
+		setOpen( false );
+		createSuccessNotice( 'Renamed Navigation menu', {
+			type: 'snackbar',
+		} );
+	};
+	const handleChange = ( title ) =>
+		editEntityRecord( 'postType', postType, postId, { title } );
+	const handleDelete = () => {
+		deleteEntityRecord( 'postType', postType, postId, { force: true } );
+		createSuccessNotice( 'Deleted Navigation menu', {
+			type: 'snackbar',
+		} );
+		goTo( '/navigation' );
+	};
+	const handleDuplicate = async () => {
+		const savedRecord = await saveEntityRecord( 'postType', postType, {
+			title: menuTitle + __( ' (Copy)' ),
+			content: navigationMenu?.content?.raw,
+			status: 'publish',
+		} );
+		if ( savedRecord ) {
+			createSuccessNotice( 'Duplicated Navigation menu', {
+				type: 'snackbar',
+			} );
+			goTo( `/navigation/${ postType }/${ savedRecord.id }` );
+		}
+	};
+
+	const element = useSelect(
```

Comment:
I think we should rename `element` here to represent the concept. 

```suggestion
	const entity = useSelect(
```